packages/teams-js/src/private/files.ts

```diff
@@ -29,18 +31,48 @@
     Sharefile = 'SHAREFILE',
     GoogleDrive = 'GOOGLEDRIVE',
     Egnyte = 'EGNYTE',
+    SharePoint = 'SharePoint',
```

Comment:
Good question. If this should change, I wonder if we should keep it consistent with the v1 content(and just fix it when v1 fixes it) or just change it in v2. I think this is a good point to think about protocol for stuff like this since I think it also depends on the importance of the change @charpercyr @erinha